### PR TITLE
spec: Fix build missing "devel" package

### PR DESCRIPTION
**Pull request by okurz**
Doing "%ifarch" in noarch packages seems to be problematic.

### CODE REVIEW

File: openQA.spec [hunk 1]
```diff
@@ -71,10 +71,8 @@
 %endif
 # The following line is generated from dependencies.yaml
 %define devel_no_selenium_requires %build_requires %qemu %test_requires curl perl(Devel::Cover) perl(Devel::Cover::Report::Codecov) perl(Perl::Tidy) postgresql-devel rsync sudo tar xorg-x11-fonts
-%ifarch x86_64 aarch64
 # The following line is generated from dependencies.yaml
 %define devel_requires %devel_no_selenium_requires chromedriver
```
Comment (Martchus) [inline openQA.spec:75]: What about keeping the `%ifarch` but adding an

```
%else
%define devel_requires %devel_no_selenium_requires
```

Comment (Martchus) [reply openQA.spec:75]: Forget my comment. Using `%ifarch` makes no sense when building a noarch package. We needed to make it an arch-specific package as soon if the make arch-specific things (which would actually be an option to consider).

Comment (okurz) [reply openQA.spec:75]: Actually I think the approach in this PR without any exclusion is correct as the dependency for a complete devel environment simply *is* including chromedriver. If we can not fulfill dependencies on the side of OBS then this is a problem that can be handled there accordingly, e.g. disable building/publishing of the subpackage.

**Comment by okurz** [comment]
Ok, circle CI tests won't help us for spec changes. OBS package build fails in https://build.opensuse.org/package/live_build_log/devel:openQA:TestGithub:PR-3251/openQA/openSUSE_Tumbleweed/x86_64 due to "./t/api/14-plugin_obs_rsync_async.t" recorded in https://progress.opensuse.org/issues/68836 and Leap:15.2 fails due to missing perl-Mojo-IOLoop-ReadWriteProcess which should be solved for the next round after I did `osc linkpac openSUSE:Factory perl-Mojo-IOLoop-ReadWriteProcess devel:openQA:Leap:15.2`

**Pull request merged**